official/resnet/resnet_run_loop.py

```diff
@@ -260,19 +280,21 @@ def exclude_batch_norm(name):
 
     optimizer = tf.train.MomentumOptimizer(
         learning_rate=learning_rate,
-        momentum=momentum)
-
-    # If we are running multi-GPU, we need to wrap the optimizer.
-    if multi_gpu:
-      optimizer = tf.contrib.estimator.TowerOptimizer(optimizer)
+        momentum=momentum
+    )
 
     update_ops = tf.get_collection(tf.GraphKeys.UPDATE_OPS)
     train_op = tf.group(optimizer.minimize(loss, global_step), update_ops)
   else:
     train_op = None
 
-  accuracy = tf.metrics.accuracy(
-      tf.argmax(labels, axis=1), predictions['classes'])
+  if not distribute_lib.has_distribution_strategy():
+    accuracy = tf.metrics.accuracy(
+        tf.argmax(labels, axis=1), predictions['classes'])
+  else:
+    # Metrics are currently not compatible with distribution strategies
+    accuracy = (tf.no_op(), tf.constant(0))
```

Comment:
Please comment noting that this is just for training metrics, and has not been found to affect performance at least for this Resnet model.